src/ui/map.js

```diff
@@ -1409,6 +1409,25 @@ class Map extends Camera {
         return this._update();
     }
 
+    /**
+     * Removes feature state, setting it back to the default behavior. If only
+     * source is specified, removes all states of that source. If
+     * target.id is also specified, removes all keys for that feature's state.
+     * If target.key is also specified, removes that key from that feature's state.
```

Comment:
` removes that key from that feature's state.` => ` removes the specified key from the target feature's state.`